regex-ballerina/natives.bal

```diff
@@ -17,7 +17,7 @@
 import ballerina/jballerina.java;
 
 
-# Checks whether the given string matches the provided regex.
+# Checks whether the given string matches the provided `regex`.
```

Comment:
In some places we used `regular expressions` and in some places, we use `regex`. can you check and make them consistent

Reply:
Changed to regex